src/check/parse/IR.zig

```diff
@@ -845,12 +863,17 @@ pub const NodeStore = struct {
                 node.region = u.region;
             },
             .alternatives => |a| {
-                std.debug.assert(a.patterns.span.len > 1);
+                // disabled because it was hit by a fuzz test
+                // for a repro see src/snapshots/fuzz_crash_012.txt
+                // std.debug.assert(a.patterns.span.len > 1);
```

Comment:
Not for this PR, but we should think about ways to leave in asserts while still documenting fuzzer failures.

Cause I don't think the correct answer to a fuzzer failure on an assert is to disable the assert. Like I don't want to make this a general practice. I think it is a bad precedence.